### DISCUSSION TITLE
Getting an error when I created prisma demo server because I put in an incorrect service name and I'm having trouble changing it.

### DISCUSSION THREAD

**Discussion by stefanosu** [prisma/prisma#4856] [Q&A]
I'm new to prisma. I tried manually logging in to change it but not able to do so. Thanks in advance. 

` environment { needsAuth: true } +0ms
  client Trying manual login +0ms
  client Error: dev%20 is an invalid service name. It must be less than 140 characters and can only contain alphanumeric characters and -.: {"response":{"data":{"generateClusterToken":null},"errors":[{"message":"dev%20 is an invalid service name. It must be less than 140 characters and can only contain alphanumeric characters and -.","locations":[{"line":3,"column":9}],"path":["generateClusterToken"],"code":222}],"status":200},"request":{"query":"\n      mutation ($input: GenerateClusterTokenRequest!) {\n        generateClusterToken(input: $input) {\n          clusterToken\n        }\n      }\n    ","variables":{"input":{"workspaceSlug":"stefanos-65e1fd","clusterName":"prisma-us1","serviceName":"dev%20","stageName":"dev"}}}} +0ms
Error: dev%20 is an invalid service name. It must be less than 140 characters and can only contain alphanumeric characters and -.: {"response":{"data":{"generateClusterToken":null},"errors":[{"message":"dev%20 is an invalid service name. It must be less than 140 characters and can only contain alphanumeric characters and -.","locations":[{"line":3,"column":9}],"path":["generateClusterToken"],"code":222}],"status":200},"request":{"query":"\n      mutation ($input: GenerateClusterTokenRequest!) {\n        generateClusterToken(input: $input) {\n          clusterToken\n        }\n      }\n    ","variables":{"input":{"workspaceSlug":"stefanos-65e1fd","clusterName":"prisma-us1","serviceName":"dev%20","stageName":"dev"}}}}
    at GraphQLClient.<anonymous> (/usr/local/lib/node_modules/prisma/node_modules/graphql-request/src/index.ts:72:13)
    at step (/usr/local/lib/node_modules/prisma/node_modules/graphql-request/dist/src/index.js:40:23)
    at Object.next (/usr/local/lib/node_modules/prisma/node_modules/graphql-request/dist/src/index.js:21:53)
    at fulfilled (/usr/local/lib/node_modules/prisma/node_modules/graphql-request/dist/src/index.js:12:58)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:84284) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
  output Exiting with code: 1 +0ms
`

**Comment by ryands17** [comment]
Hey @stefanosu 👋 

It seems that you are following a tutorial that's related to an older version of Prisma. I would strongly suggest starting with Prisma 2 and following the steps described [here](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch-typescript-postgres) as Prisma 1 is no longer maintained and will not get any new features.

Also it would be great if you could point to the tutorial that you were following so that we can deprecate it :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
